### PR TITLE
Fixed some Profile Serialization Issues and added Quickstart Menu

### PR DESCRIPTION
**Pull request by StephenHodgson**
- We were having issues with active profiles being registered to the Mixed Reality Manager whenever we crated a new scene and selected a config profile.
- I also added a Quickstart menu under Mixed Reality Toolkit menu to generate the Mixed Reality Manager in the scene if none exists.
- Updated how we create profiles.
- Updated Camera Profile Inspector to match the rest of the profiles.
- Fixed Profile Activation crashing if it's null or unset.

### CODE REVIEW

File: Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs [hunk 1]
```diff
@@ -218,7 +218,7 @@ void ISerializationCallbackReceiver.OnAfterDeserialize()
         public MixedRealityControllerMapping GetControllerMapping(Type controllerType, Handedness hand)
         {
             var systemType = SystemType.GetReference(controllerType);
-            for (int i = 0; i < ControllersProfile?.MixedRealityControllerMappingProfiles.Length; i++)
```
[the remark this answers is not in the document]
Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs]: Yes bc ControllersProfile is a unity engine type that can never be null.

[the remark this answers is not in the document]
Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs]: It's for the same reason why you can't use the `??` on Unity.Object types.

[the remark this answers is not in the document]
Comment (keveleigh) [reply Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs]: Should we add an `== null` check then instead? If I delete the ControllersProfile from the ConfigurationProfile, Unity crashes on play.

[the remark this answers is not in the document]
Comment (keveleigh) [reply Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs]: It also looks like unchecking `EnableControllerProfiles` doesn't actually do anything except hide the field in the Inspector (if the field is filled in before unchecking, it still runs and works), but that's unrelated to this PR.

[the remark this answers is not in the document]
Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs]: Also, as far as I'm aware, we're not even using the Controller Profile yet. It's not wired up to anything.

[the remark this answers is not in the document]
Comment (keveleigh) [reply Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs]: It disables the bool, but the bool doesn't seem to be used anywhere except to decide whether to display the field in the Inspector. As long as `controllersProfile` has a value before the field disappears, `GetControllerMapping` still uses it.

[the remark this answers is not in the document]
Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs]: Right, we haven't wired it up anywhere yet.

[the remark this answers is not in the document]
Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs]: I'm not even really sure what that GetControllerMapping is for anyway. Gotta ask @SimonDarksideJ

[the remark this answers is not in the document]
Comment (SimonDarksideJ) [reply Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs]: yes it was being used., That is until @StephenHodgson deleted the code in my devices branch :P 

But, as to the Bool, I think that is currently only used by the Editor.  Will have to double check and validate in the controller space, that we test this before loading profiles.

This bool would / should also effectively disable controllers.   No config, no controllers :D

**Pull request merged**